dq0/sdk/data/utils/util.py

```diff
@@ -447,39 +447,43 @@ def get_percentage_freq_of_values(x_np_a):
 
 def estimate_freq_of_labels(y):
     """Estimate the frequency of labels in y."""
+    if isinstance(y, pd.DataFrame):
+        y = y.idxmax(axis=1)
     if isinstance(y, pd.Series):
-        # print(y.value_counts(normalize=True) * 100)
-        y = y.values
-    # else:
-    assert isinstance(y, np.ndarray)
-
+        pass
+    elif isinstance(y, np.ndarray):
+        if y.ndim == 2:
+            if len(y[0]) > 1:
+                y = np.argmax(y, axis=1)
+            else:
+                y = np.ravel(y)
+        y = pd.Series(y)
+    else:
+        raise ValueError('y must be either pd.DataFrame, pd.Series or np.ndarray')
+    
     for key, value in get_percentage_freq_of_values(y).items():
         msg = '  label "{}": {:.1f}%'.format(key, value)
         print(msg)
 
 
-# def compute_features_min_max(X):
+# def compute_features_bounds(X):
```

Comment:
Minor thing: since it actually computes min/max rather than any loose upper/lower bound, I'd keep the name min_max

Reply:
fine. missed the name change on merge.